### DISCUSSION TITLE
Qselect multiple values

### DISCUSSION THREAD

**Discussion by BalazsMakrai** [quasarframework/quasar#14851] [General - Components / Directives / etc]
I'm trying to add a Qselect with multiple values. I want to load some predefined values, but they are not marked as selected, so I can add them again.

Here is the screenshot:
![dfdgfdgfdgfdfg](https://user-images.githubusercontent.com/64318333/200541358-d7089293-b3d8-4541-b3c9-e08753179e8d.jpg)

The code is:
 `<q-select outlined multiple v-model="props.row.assignedUsers" :options="assignedSelect"></q-select>`

Is there any way to prevent adding the already existing value?

**Comment by suleimanas** [comment]
Do you mean hide-selected
